mmdet/models/dense_heads/base_dense_head.py

```diff
@@ -14,10 +17,181 @@ def loss(self, **kwargs):
         """Compute losses of the head."""
         pass
 
-    @abstractmethod
-    def get_bboxes(self, **kwargs):
+    @force_fp32(apply_to=('cls_scores', 'bbox_preds'))
```

Comment:
It is necessary to add a detailed docstring here

Reply:
I added a description, but the English expression ability is relatively poor, you can help to change it.